src/utils/get-authorization-url.ts

```diff
@@ -13,6 +13,7 @@ export function getAuthorizationUrl(
 		redirectUri,
 		state,
 		screenHint,
+    invitationToken,
```

Comment:
this file needs to get formatted (it's tabs ??), but i'll do that in a separate PR i think.